packages/blade/src/components/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import { Drawer, DrawerBody, DrawerHeader } from './';
+import type { DrawerProps } from './';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import { Box } from '~components/Box';
+import iconMap from '~components/Icons/iconMap';
+import { Button } from '~components/Button';
+import { AnnouncementIcon, DownloadIcon } from '~components/Icons';
+import { Card, CardBody } from '~components/Card';
+import { Badge } from '~components/Badge';
+
+export default {
+  title: 'Components/Drawer',
+  component: Drawer,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+    icon: {
+      name: 'icon',
+      type: 'select' as 'string',
```

Comment:
huh?

Reply:
Remember we discussed this with Kamlesh last time? storybook types mess. We have to do this at all places otherwise it throws error in storybook.

Anyway you can ignore it since this is temporary story I copy pasted. This prop itself won't exist in actual storybook